app/services/import/legacy/fetch-licence-holders.service.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+/**
+ * Fetches the Licence Holder data from the import.NALD_PARTIES table for the licence ref
+ * @module FetchLicenceHolderService
+ */
+
+const { db } = require('../../../../db/db.js')
+
+/**
+ * Fetches the licence holder data from the import.NALD_PARTIES table for the licence ref
+ *
+ * The licence holder is determined from the import.NALD_ABS_LIC_VERSIONS table. We get the earliest start date
+ *
+ * @param {string} regionCode - The NALD region code
+ * @param {string} licenceId - The NALD licence ID
+ *
+ * @returns {Promise<ImportLegacyLicenceHolderType[]>}
+ */
+async function go (regionCode, licenceId) {
+  const query = _query()
+
+  const { rows } = await db.raw(query, [regionCode, licenceId])
+
+  return rows
+}
+
+function _query () {
+  return `
+    SELECT DISTINCT ON (np."ID")
+      (concat_ws(':', np."FGAC_REGION_CODE", np."ID")) AS company_external_id,
+      (concat_ws(':', np."FGAC_REGION_CODE", np."ID")) AS contact_external_id,
+      to_date(nalc."EFF_ST_DATE", 'DD/MM/YYYY') as start_date,
```

Comment:
Where is the returns communication type coming from - double check this is done elsewhere.